modules/actor/actor-wfrp4e.js

```diff
@@ -894,6 +894,16 @@ export default class ActorWfrp4e extends Actor {
     mergeObject(testData, this.getPrefillData("cast", spell, options))
 
 
+    //@HOUSE
+    testData.unofficialGrimoire = game.settings.get("wfrp4e", "unofficialgrimoire");
+    let advantages = this.status.advantage.value || 0;
+    if (testData.unofficialGrimoire) {
+      advantages = "N/A";
+    } else {
+      this.status.advantage.value || 0
+    }
+    //@HOUSE
```

Comment:
Can you add  `game.wfrp4e.utility.logHomebrew("unofficialgrimoire")` inside the true block?

Reply:
done